instrumentation/opentelemetry-instrumentation-django/README.rst

```diff
@@ -15,52 +15,6 @@ Installation
 
     pip install opentelemetry-instrumentation-django
 
```

Comment:
Where did this one go?

Reply:
It's already in [here](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py#L14)